generators/client/templates/angular/webpack/webpack.prod.js.ejs

```diff
@@ -113,7 +113,10 @@ module.exports = webpackMerge(commonConfig({ env: ENV }), {
                 parallel: true,
                 cache: true,
                 terserOptions: {
+                    ecma: 6,
                     ie8: false,
+                    toplevel: true,
+                    module: true,
```

Comment:
why are we adding those two options ?

Reply:
Docs:

> module (default false) — Use when minifying an ES6 module. "use strict" is implied and names can be mangled on the top scope. If compress or mangle is enabled then the toplevel option will be enabled.
toplevel (default false) - set to true if you wish to enable top level variable and function name mangling and to drop unused variables and functions.